src/azure-cli/azure/cli/command_modules/appconfig/_help.py

```diff
@@ -121,6 +125,8 @@
     text: az appconfig kv export -n MyAppConfiguration -d appconfig --dest-name AnotherAppConfiguration --key * --label * --preserve-labels
   - name: Export all keys and feature flags with all labels to another App Configuration and overwrite destination labels.
     text: az appconfig kv export -n MyAppConfiguration -d appconfig --dest-name AnotherAppConfiguration --key * --label * --dest-label ExportedKeys
+  - name: Export all keys to another App Configuration using  your 'az login' credentials.
+    text: az appconfig kv export -d appconfig -n MyAppConfiguration --auth-mode login --dest-name AnotherAppConfiguration --dest-auth-mode login --key * --label * --preserve-labels
```

Comment:
nit: can we use endpoint in our example for aad auth? Otherwise, the command is more likely to fail and we have to explain the login has to have both ARM and app configuration role.

Reply:
sure